cmd/kev/cmd/init.go

```diff
@@ -88,25 +92,53 @@ func runInitCmd(cmd *cobra.Command, _ []string) error {
 		return displayInitError(err)
 	}
 
-	manifest, err := kev.Init(files, envs, ".")
+	manifest, skaffoldManifest, err := kev.Init(files, envs, ".", skaffold)
 	if err != nil {
 		return displayInitError(err)
 	}
 
```

Comment:
Funnily enough it started like this but since we map command to a matching action I encapsulated everything in Init. I don't necessarily like it and yeah I think I revert it back to what it was originally...